apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -104,6 +105,12 @@ export default function sectionAssessments(state=initialState, action) {
       ...initialState
     };
   }
+  if (action.type === SET_SCRIPT) {
+    return {
+      ...state,
+      studentId: ALL_STUDENT_FILTER,
```

Comment:
nice -- I hadn't seen this pattern before, where a reducer listens for events owned by a different reducer. looks good!

Reply:
Thanks! we've done it a few times in teacher dashboard because we have a few modules that are "data/business logic" and shared between tabs, and then we have the "UI/tab specific" modules that sometimes rely on changes from those shared modules.